dockerfiles/docker-compose-webpack.yml

```
@@ -0,0 +1,27 @@
+# docker-compose-webpack.yml starts Webpack dev server for hot reload of built
+# static assets.
+version: '3'
+
+services:
+
+  webpack:
+    image: node:12
+    working_dir: /usr/src/app/checkouts/ext-theme
+    environment:
+      - NODE_ENV=development
+    ports:
+      - "10001:10001"
+    volumes:
+      - ${PWD}/common/dockerfiles/entrypoints/common.sh:/usr/src/app/docker/common.sh
+      - ${PWD}/common/dockerfiles/entrypoints/webpack.sh:/usr/src/app/docker/webpack.sh
+      - ${PWD}/${RTDDEV_PATH_EXT_THEME:-../readthedocs-ext-theme}:/usr/src/app/checkouts/ext-theme
```

Believe this is ext-theme now?

```suggestion
      - ${PWD}/${RTDDEV_PATH_EXT_THEME:-../ext-theme}:/usr/src/app/checkouts/ext-theme
```

---

it seems that you lost this with the rebase, which makes my local env to fail

---

Oops, let's try this again!